libs/gi/db/src/Database/DataManagers/TeamCharacterDataManager.ts

```diff
@@ -112,10 +112,10 @@ export class TeamCharacterDataManager extends DataManager<
     )
     for (
       let num = existingUndercKey.length + 1;
-      num < existingUndercKey.length + 100;
+      num <= existingUndercKey.length + 100;
```

Comment:
nit: still don't need this +100 stuff. going from `length+1` to `length*2` guarantees that one of those values will succeed